VAMobile/src/screens/HealthScreen/SecureMessaging/Inbox/Inbox.tsx

```diff
@@ -60,7 +60,7 @@ const Inbox: FC<InboxProps> = () => {
 
   return (
     <Box {...testIdProps('', false, 'Inbox-page')}>
-      <MessageList items={getMessagesListItems(inboxMessages || [], t, onInboxMessagePress)} title={t('secureMessaging.inbox')} />
+      <MessageList items={getMessagesListItems(inboxMessages || [], t, onInboxMessagePress, 'Inbox')} title={t('secureMessaging.inbox')} />
```

Comment:
Mainly curious, was this added for readability reasons? Also, we can use the [FolderNameTypeConstants](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/f8424b2aa45666c823eb836e4b034c64251bdb0c/VAMobile/src/constants/secureMessaging.ts#L19-L29) here, so we don't have to hard-code the folder name